json_rpc/http.md

```diff
@@ -0,0 +1,339 @@
+# JSON RPC HTTP Methods
```

Comment:
We should add in something about auth. We're gonna move to a world where roughly:

1. Unsigned requests are low priority
2. Signed requests are medium priority
3. Signed requests with good reputation are highest priority

In the short-term we'll just require requests to be signed by a set of white listed auth keys. Clients will need to sign the request body with their whitelisted ed25519-dalek keypair and add the signature to their http headers under the keyname "authorization".